.github/workflows/build.yml

```
@@ -28,34 +29,37 @@ on:
 
```

```suggestion
permissions: {}

```
https://woodruffw.github.io/zizmor/audits/#remediation_2
```
warning[excessive-permissions]: overly broad permissions
   --> .github/workflows/build.yml:1:1
    |
  1 | / name: Build
  2 | |
...   |
220 | |           echo "### :tada: Checks Passed!" >> "${GITHUB_STEP_SUMMARY}"
221 | |           echo "" >> "${GITHUB_STEP_SUMMARY}"
    | |______________________________________________- default permissions used due to no permissions: block
    |
    = note: audit confidence → Medium
```

---

I think you are using `-p` here right?
Those are not really an issue i think, see https://github.com/woodruffw/zizmor/issues/472

---

Yeah, forgot to mention that.